src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/importer/price/PriceImporter.kt

```diff
@@ -20,18 +22,18 @@ class PriceImporter(
 
   private val logger = LoggerFactory.getLogger(javaClass)
 
-  fun import(supplier: Supplier) {
+  fun import(supplier: Supplier, timeSource: TimeSource) {
```

Comment:
Sorry, TimeSource should be wired in via the constructor, it is only done once then and we don't need to keep passing it in, so....

`class PriceImporter(private val timeSource: TimeSource, .....)`